deployment/ansible/roles/ashlar.database/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Create PostgreSQL super user
+  sudo_user: postgres
+  postgresql_user: name="{{ postgresql_username }}"
+                   password="{{ postgresql_password }}"
+                   role_attr_flags=SUPERUSER
+                   state=present
+
+- name: Create PostgreSQL database
+  sudo_user: postgres
+  postgresql_db: name="{{ postgresql_database }}"
+                 owner="{{ postgresql_username }}"
+
+- name: Add PostGIS extension
```

Comment:
Ansible 1.9 has a module to do this now. I haven't used it, but if you want to enforce 1.9+, you can give it a try.